server/routes/metrics/metrics_rounter.ts

```diff
@@ -62,4 +70,153 @@ export function registerMetricsRoute(router: IRouter) {
       }
     }
   );
+
+  router.get(
+    {
+      path: `${OBSERVABILITY_BASE}/search/indices`,
+      validate: {},
+    },
+    async (context, request, response) => {
+      const indexPattern = 'ss4o_metrics-*-*';
```

Comment:
can use the constant here

Reply:
I haven't declared this constant anywhere else, this is the only place its used

Reply:
Then I think this is a good candidate for constant 😄

Reply:
used a constant :)